src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferMetadataV1.java

```diff
@@ -50,7 +51,10 @@ public SnowflakeFileTransferMetadataV1(
 
   @Override
   public boolean isForOneFile() {
-    return stageInfo.getStageType() == StageInfo.StageType.GCS;
+    // The presigned url is for one file and the down-scoped token can be used for multiple files.
+    return stageInfo.getStageType() == StageInfo.StageType.GCS
```

Comment:
I suggest to change the name of this method to reflect that it checks for presigned URL stuff. so something like isForGCSPresignedURL().

Reply:
I prefer to keep the name. The main reason is the old version of SC is using `isForOneFile()`. 
If we rename it, the new JDBC may not work with all released JDBC even if they are not using GCP.